tests/integration/modules/test_win_lgpo.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import os
+import textwrap
+import re
+import io
+
+# Import Salt Testing libs
+from tests.support.case import ModuleCase
+from tests.support.unit import skipIf
+from tests.support.helpers import destructiveTest, generate_random_name
+from tests.support.runtests import RUNTIME_VARS
+
+# Import Salt libs
+import salt.utils.files
+import salt.utils.platform
+
+@skipIf(not salt.utils.platform.is_windows(), 'windows test only')
+class WinLgpoTest(ModuleCase):
+    '''
+    Tests for salt.modules.win_lgpo
+    '''
+
+    def setUp(self):
+        '''
+        setup function
+
+        downloads and extracts the lgpo.exe tool into c:\windows\system32
+        for use in validating the registry.pol files
+        '''
+        ret = self.run_function('archive.unzip',
+                                'https://www.microsoft.com/en-us/download/confirmation.aspx?id=55319&6B49FDFB-8E5B-4B07-BC31-15695C5A2143=1',
+                                dest='c:\\windows\\system32')
```

Comment:
The `setUp` function runs before each individual test... you'll probably want to use the `LoaderModuleMockMixin`

In your imports:
```
from tests.support.mixins import LoaderModuleMockMixin
```
Then in the class add the mixin:
```
class WinLgpoTest(ModuleCase, LoaderModuleMockMixin):
```
Then rename the `setUp` function to `setup_loader_modules`:
```
    def setup_loader_modules(self):
        '''
        setup function
         downloads and extracts the lgpo.exe tool into c:\windows\system32
        for use in validating the registry.pol files
        '''
        ret = self.run_function('archive.unzip',
                                'https://www.microsoft.com/en-us/download/confirmation.aspx?id=55319&6B49FDFB-8E5B-4B07-BC31-15695C5A2143=1',
                                dest='c:\\windows\\system32')
```